lms/djangoapps/lms_xblock/runtime.py

```diff
@@ -284,4 +284,8 @@ def applicable_aside_types(self, block):
         if block.scope_ids.block_type in config.disabled_blocks.split():
             return []
 
-        return super(LmsModuleSystem, self).applicable_aside_types()
+        return [
+            aside_type
+            for aside_type in super(LmsModuleSystem, self).applicable_aside_types(block)
+            if aside_type != 'acid_aside'
```

Comment:
@cpennington could you please explain how to do this? Because as I could see the same approach with `if` statement is already used in the CMS part https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/preview.py#L129 . As I understand  `Runtime.applicable_aside_types` scans all entry points to find `xblock_asides.v1` entities. But `acid-block` is taken from the [remote repository](https://github.com/edx/edx-platform/blob/master/requirements/edx/github.txt#L77).